src/Razor/src/Microsoft.AspNetCore.Razor.VSCode/src/BlazorDebug/BlazorDebugConfigurationProvider.ts

```diff
@@ -126,8 +102,7 @@ export class BlazorDebugConfigurationProvider implements vscode.DebugConfigurati
              * We do this to provide immediate visual feedback to the user
              * that their debugger session has started.
              */
-            await this.vscodeType.debug.startDebugging(folder, browser);
-            this.logger.logVerbose('[DEBUGGER] Launching browser debugger...');

```

Comment:
Seems like you removed a bunch of the log statements, did we just find them to be unhelpful?

Reply:
Practically, no. These logs don't end up being all that helpful. We rely on errors from the underlying debuggers and the logs that the JS debugger produces for almost all our investigations.